docs: expand streaming conceptual guides (#1524)
langchain-ai/langgraph: docs/docs/cloud/how-tos/stream_updates.md

# How to stream state updates of your graph

> This guide covers how to use `stream_mode="updates"` for your graph, which will stream the updates to the graph state that are made after each node is executed. This differs from using `stream_mode="values"` because instead of streaming the entire value of the state after every node is run, it only streams the updates that node made to the state. Click [here](https://langchain-ai.github.io/langgraph/concepts/low_level/#stream-and-astream) to see a nice diagram that intuitively explains the difference.

Reviewer: ```suggestion
This guide covers how to use `stream_mode="updates"` for your graph, which will stream the updates to the graph state that are made after each node is executed. This differs from using `stream_mode="values"`: instead of streaming the entire value of the state at each superstep, it only streams the updates from each of the nodes that made an update to the state at that superstep. Click [here](https://langchain-ai.github.io/langgraph/concepts/low_level/#stream-and-astream) to see a nice diagram that intuitively explains the difference.
```
